### DISCUSSION TITLE
Update CVEs

### DISCUSSION THREAD

**Discussion by ConTayl** [telekom-security/tpotce#1333] [General]
Hi, so I've noticed that my T-Pot results are quite outdated and there aren't many of them. If I'm right the rules are pulled from emerging threats. I'd just wondered if anybody knows how I could increase the amount of CVE's as well as increasing the amount of new ones. 
Apologies if this is slightly unclear any help is appreciated.

**Comment by t3chn0m4g3** [comment]
What do you mean by outdated? T-Pot can only classify the events attackers actually run against your installation of T-Pot.
